src/TVar.h

```diff
@@ -74,4 +78,40 @@ class TVar
     QString nName;
 };
 
+#ifndef QT_NO_DEBUG_STREAM
+inline QDebug& operator<<(QDebug& debug, const TVar* var)
+{
+    QDebugStateSaver saver(debug);
+    Q_UNUSED(saver);
+    debug.nospace() << "TVar(" << var->getName() << ")";
+    switch (var->getKeyType()) {
+        case LUA_TNONE: debug.nospace() << ", keyType=none"; break;
+        case LUA_TNIL: debug.nospace() << ", keyType=nil"; break;
+        case LUA_TBOOLEAN: debug.nospace() << ", keyType=boolean"; break;
+        case LUA_TLIGHTUSERDATA: debug.nospace() << ", keyType=lightuserdata"; break;
+        case LUA_TNUMBER: debug.nospace() << ", keyType=number"; break;
+        case LUA_TSTRING: debug.nospace() << ", keyType=string"; break;
+        case LUA_TTABLE: debug.nospace() << ", keyType=table"; break;
+        case LUA_TFUNCTION: debug.nospace() << ", keyType=function"; break;
+        case LUA_TUSERDATA: debug.nospace() << ", keyType=userdata"; break;
+        case LUA_TTHREAD: debug.nospace() << ", keyType=thread"; break;
+    }
+    switch (var->getValueType()) {
+        case LUA_TNONE: debug.nospace() << ", valueType=none"; break;
+        case LUA_TNIL: debug.nospace() << ", valueType=nil"; break;
+        case LUA_TBOOLEAN: debug.nospace() << ", valueType=boolean"; break;
+        case LUA_TLIGHTUSERDATA: debug.nospace() << ", valueType=lightuserdata"; break;
+        case LUA_TNUMBER: debug.nospace() << ", valueType=number"; break;
+        case LUA_TSTRING: debug.nospace() << ", valueType=string"; break;
+        case LUA_TTABLE: debug.nospace() << ", valueType=table"; break;
+        case LUA_TFUNCTION: debug.nospace() << ", valueType=function"; break;
+        case LUA_TUSERDATA: debug.nospace() << ", valueType=userdata"; break;
+        case LUA_TTHREAD: debug.nospace() << ", valueType=thread"; break;
+    }
+    debug.nospace() << ')';
+    return debug;
+}
```

Comment:
I note that no attempt is make to display the data where that is fesible - is that intended?

Reply:
What do you mean by this?

Reply:
You're right, I'll fix.

Reply:
Fixed.